README.md

```diff
@@ -1,3 +1,5 @@
+**NOTICE** This fork addresses this [open issue](https://github.com/fog/fog-google/issues/590) with `fog-google` that make newer versions of the gem incompatible with our `rma` app. If that issue is fixed, we can stop using this fork and get back on the official gem. We should also pull in upstream changes into this fork when possible.
```

Comment:
Nice comment! We should do something loud like this on each one of our other unfortunate forks, ha.